_posts/2020-10-21-explicit-term-inference-in-scala-3.md

```diff
@@ -0,0 +1,430 @@
+---
+layout: blog-detail
+post-type: blog
+by: Meriam Lachkar, Vincenzo Bazzucchi, Scala Center
+title: Explicit term inference with Scala 3
+---
+
+# Explicit term inference with Scala 3
+
+One of the most striking changes for developers adopting Scala 3 is the introduction
+of new syntaxes to replace the implicit mechanism used in previous Scala versions.
```

Comment:
"new syntaxes and semantics", perhaps?

Reply:
Or just "new syntax"? I wouldn't normally _count_ syntax(es).